src/packages/console/src/pages/createSitePage.tsx

```diff
@@ -143,99 +121,94 @@ export const CreateSitePage = (): React.ReactElement => {
+        <NavigationBar />
+        <Stack direction={Direction.Vertical} paddingVertical={PaddingSize.Wide2} isFullHeight={true}>
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card'>
+            <Stack direction={Direction.Vertical} shouldAddGutters={true}>
+              <Stack.Item alignment={Alignment.Center} gutterAfter={PaddingSize.Wide2}>
+                <Text tag='h1' variant='header5'>Create a new site</Text>
+              </Stack.Item>
+              {accounts === null ? (
+                <Text variant='note'>An error occurred. Please try again later.</Text>
+              ) : (
+                <Form isLoading={isLoading || accounts === undefined} onFormSubmitted={onCreateSiteClicked}>
+                  <Stack direction={Direction.Vertical} shouldAddGutters={true} isFullWidth={true}>
+                    <FormControl
+                      variant='outlined'
+                      margin='normal'
+                      required
+                      fullWidth
+                    >
+                      <InputLabel id='account-select-label'>Account</InputLabel>
+                      <Select
+                        labelId='account-select-label'
+                        value={selectedAccountId}
+                        onChange={onAccountSelected}
+                        error={selectedAccountIdError !== undefined}
+                        label='Account'
+                        // helperText={selectedAccountIdError}
+                      >
+                        {accounts?.map((account: Account, index: number): React.ReactElement => (
+                          <MenuItem key={index} value={account.accountId}>{account.name}</MenuItem>
+                        ))}
+                      </Select>
+                    </FormControl>
+                    <SingleLineInput
+                      id='slug'
+                      label='Site slug'
+                      name='slug'
+                      value={slug}
+                      onValueChanged={onSlugChanged}
+                      inputType={InputType.Text}
+                      inputWrapperVariant={slugError ? 'error' : 'default'}
+                      messageText={slugError || (!slug ? 'This will be your everypage sub-domain e.g. hello.evrpg.com' : `Your everypage sub-domain will be ${slug}.evrpg.com`)}
+                      placeholderText='Site slug'
+                    />
+                    <SingleLineInput
+                      id='name'
+                      label='Site name'
+                      name='name'
+                      value={name}
+                      onValueChanged={onNameChanged}
+                      inputType={InputType.Text}
+                      inputWrapperVariant={nameError ? 'error' : ''}
+                      messageText={nameError}
+                      placeholderText='Site name'
+                    />
+                    <StyledBox bordered>
```

Comment:
does this not work if you just use `Box variant='bordered'>`?

Reply:
I tried using box but I was not able to reduce the padding in the box and also I was not able to give a left gutter when I was using box.

Reply:
the gutter can go on the stack and you can use `unpadded` i think? not totally sure but worth trying i think?

Reply:
I tried that, didn't work the way I wanted it to

Reply:
ok cool

Reply:
I have removed the comments

Reply:
Now just the select is using material-ui
So, if want I can try to make a dropdown select for ui-react